float_data_for_tock.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import os
+import json
+
+api_key = os.environ.get('FLOAT_API_KEY')
+headers = {'Authorization': 'Bearer ' + api_key}
+endpoint = 'tasks'
+year = "2016"
+month = "04"
+day = "01"
+weeks = "52"
```

Comment:
static values here could be replaced by dynamic variables to generate a smaller data set.